.github/workflows/appengine-deploy.yml

```diff
@@ -0,0 +1,23 @@
+name: appengine-deploy Unit
+on: [push, pull_request]
+jobs:
+  run:
+    name: appengine-deploy
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: [ubuntu-latest, windows-latest, macos-latest]
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-node@master
+      with:
+        node-version: 12.x
+
+    - name: npm install
+      run: npm install
+      working-directory: ./appengine-deploy
+
+    - name: npm lint
```

Comment:
Should also include a npm test invocation.

Reply:
I don't have any unit tests, only the integration tests. The setupGcloudSDK has all the unit tests. Do you have guidance on adding unit tests?

Reply:
Gotcha. As long as the logical units have test coverage, housing them in setupGcloudSDK makes sense.